Dockerfile

```diff
@@ -121,7 +121,8 @@ FROM scratch as workflow-controller
 
 USER 8737
 
-COPY --from=workflow-controller-build /usr/share/zoneinfo /usr/share/
+ADD --chown=8737 https://github.com/golang/go/raw/master/lib/time/zoneinfo.zip /zoneinfo.zip
+ENV ZONEINFO /zoneinfo.zip
```

Comment:
We used to load these binaries from the base image... why do we need to get them from github?

Reply:
this is what was recommended in the Stackoverflow issue, and it works